### PR TITLE
Bring PR information inline with confluence

### PR DESCRIPTION
**Pull request by BeccaAnderton**
## Overview
- [x] I have performed a self review of my code
- [x] I have made corresponding changes to the documentation
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added sufficient logging
- [x] New and existing tests pass locally with my changes
- [x] The code modified as part of this PR has been covered with tests

### Description
As discussed there were some discrepancies between the pull request information on code standards and confluence so I've tried to bring them inline and match. 

### Dependencies
N/A

### Test procedures
N/A

### Links

Confluence: https://netsells.atlassian.net/wiki/spaces/NO/pages/938442753/Pull+Request+Strategy

### CODE REVIEW

File: src/general/pull-requests.md [hunk 1]
```diff
@@ -19,11 +25,32 @@ The majority of projects have automated test suites and linters which are run by
...
+When leaving a review, we have two types of pull request comments, **MUST** and **SHOULD**, these keywords to indicate the level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know. 
 
 Comments should be critical, informative and descriptive enough for you to implement the suggestion. Comments such as “this is wrong” or “this doesn’t work” are not acceptable and should be replaced with an explanation of the problem and suggestions for the solution, including code snippets where appropriate.
 
-Once you have made the requested changes, you can re-request a review from a user. The person who left the original comment will mark the comment as resolved if they are happy with the solution.
+
```
Comment (samturrell) [inline src/general/pull-requests.md:32]: Lots of white space here

File: src/general/pull-requests.md [hunk 2]
```diff
@@ -19,11 +25,32 @@ The majority of projects have automated test suites and linters which are run by
...
 
 ## Review Comments
-When leaving a review, we use the **MUST** and **SHOULD** keywords to indicate the level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know. 
+When leaving a review, we have two types of pull request comments, **MUST** and **SHOULD**, these keywords to indicate the level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know.
```
Comment (samturrell) [inline src/general/pull-requests.md:28]: First sentence doesn't quite read right

File: src/general/pull-requests.md [hunk 3]
```diff
@@ -19,11 +25,29 @@ The majority of projects have automated test suites and linters which are run by
...
 
 ## Review Comments
-When leaving a review, we use the **MUST** and **SHOULD** keywords to indicate the level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know. 
+When leaving a review, we use two types of pull request comments, **MUST** and **SHOULD**, these keywords will indicate the requirement level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know.
```
Comment (JBonwick) [inline src/general/pull-requests.md:28]: Suggestions aren't quite as nice in a .md file! We have a lot of commas in this sentence- maybe splitting it into two would read better?
```suggestion
When leaving a review, we use two types of pull request comments; **MUST** and **SHOULD**. These keywords will indicate the requirement level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know. 
```

File: src/general/pull-requests.md [hunk 4]
```diff
@@ -19,11 +25,29 @@ The majority of projects have automated test suites and linters which are run by
...
+'user' => [
+    'first_name', 
+    'last_name',
+],
+```
+
+## Comment Tips and Examples for Reviewers
+- Try to give clear instructions on what you want changing and do not refer to in-person discussions. Pull requests are often looked back on to understand why decisions were made, "as we spoke about on Tuesday" does not help anyone 5 months later!
+- If an in-person discussion is needed (ie it'll be a lengthy comment thread) - make sure you post back with the summary so it's known to others
+- Provide clear examples whenever possible to help mentor and avoid too many back and forth with clarifications.
```
Comment (JBonwick) [inline src/general/pull-requests.md:48]:
```suggestion
- Provide clear examples whenever possible to help mentor and avoid excessive back and forth with clarifications.
```

File: src/general/pull-requests.md [hunk 5]
```diff
@@ -19,11 +25,29 @@ The majority of projects have automated test suites and linters which are run by
...
+- Read every line of code
+- Don't hold author hostage - We have enough power to block a PR, lets not use this for evil by requesting changes outside the scope of the PR. e.g fixing old code
```
Comment (JBonwick) [inline src/general/pull-requests.md:51]:
```suggestion
- Add links to comments where appropriate. External docs and discussions around topics are extremely valuable learning resources.

```

File: src/general/pull-requests.md [hunk 6]
```diff
@@ -3,6 +3,12 @@
 ## Overview
 This document details what is expected of you and what you can expect during the PR review process. This guide is not definitive or exhaustive but it does cover many of the common obstacles that we all come across. Remember that the intention of a review is to ensure the code quality is as high as possible and discussion is always welcome.
 
+We run all of our code through pull requests for a variety of reasons, these include:
```
Comment (ptrbrynt) [inline src/general/pull-requests.md:6]:
```suggestion
We run all of our code through pull requests for a variety of reasons. These include:
```

File: src/general/pull-requests.md [hunk 7]
```diff
@@ -19,11 +25,30 @@ The majority of projects have automated test suites and linters which are run by
...
 
 ## Review Comments
-When leaving a review, we use the **MUST** and **SHOULD** keywords to indicate the level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know. 
+When leaving a review, we use two types of pull request comments; **MUST** and **SHOULD**. These keywords will indicate the requirement level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know.
```
Comment (ptrbrynt) [inline src/general/pull-requests.md:28]:
```suggestion
When leaving a review, we use two types of pull request comments: **MUST** and **SHOULD**. These keywords will indicate the requirement level of a comment. If a reviewer leaves a **MUST** comment, the pull request will be rejected pending changes. It is up to the reviewers discretion to decide whether a PR is rejected due to **SHOULD** comments. Comments can also be left without this syntax which could be a general question asking you for more information or a piece of information that you may not know. 
```

File: src/general/pull-requests.md [hunk 8]
```diff
@@ -19,11 +25,30 @@ The majority of projects have automated test suites and linters which are run by
...
+SHOULD: we typically split arrays onto multiple lines for better gif diffs (with a trailing comma)
+
+'user' => [
+    'first_name', 
+    'last_name',
+],
+```
+
+## Comment Tips and Examples for Reviewers
+- Try to give clear instructions on what you want changing and do not refer to in-person discussions. Pull requests are often looked back on to understand why decisions were made, "as we spoke about on Tuesday" does not help anyone 5 months later!
```
Comment (ptrbrynt) [inline src/general/pull-requests.md:46]:
```suggestion
- Try to give clear instructions on what you want changing and do not refer to in-person discussions. Pull requests are often looked back on to understand why decisions were made; "as we spoke about on Tuesday" does not help anyone 5 months later!
```

**Review by ptrbrynt — APPROVED** [review]

**Pull request merged**